src/javascript/app_2/data/dao.js

```diff
@@ -6,24 +6,41 @@ const DAO = (() => {
 
     const getContractsFor = (symbol) => BinarySocket.send({ contracts_for: symbol });
 
+    const getOauthApps = () =>  BinarySocket.send({ oauth_apps: 1 });
+
     const getPayoutCurrencies = () => BinarySocket.send({ payout_currencies: 1 });
 
+    const getPortfolio = () => BinarySocket.send({ portfolio: 1 });
+
     const getWebsiteStatus = () => BinarySocket.send({ website_status: 1 });
 
+    const sellExpired = () => BinarySocket.send({ sell_expired: 1 });
+
     // ----- Streaming calls -----
+    const subscribeProposalOpenContract = (type, cb, should_forget_first) =>
```

Comment:
Since the value of `proposal_open_contract ` is always `1`, I'd suggest to pass `contract_id` instead of `type`, then include it in the request object only if it's available.